src/main/java/frc/robot/stateManagement/PlacementMode.java

```diff
@@ -4,7 +4,8 @@
 
 public enum PlacementMode {
   SPEAKER(0),
-  AMP(10);
+  AMP(10),
+  SOURCE(20);
```

Comment:
I can see why we would want to have all of these states as within a single enumeration, but maybe `PlacementMode` is no longer an accurate name.

Reply:
Is that something we want to change in this PR?

Reply:
NodeMode?